src/auth/src/basic.rs

```diff
@@ -0,0 +1,81 @@
+use super::{Auth, AuthRequest, Role, Object, Action};
+use super::error::AuthError;
+
+use async_trait::async_trait;
+use std::collections::HashMap;
+use fluvio_service::IdentityContext;
+
+pub type Policy = HashMap<Role, HashMap<Object, Vec<Action>>>;
+
+pub struct BasicAuth {
+    role: Option<Role>,
+    policy: Policy,
+}
+
+impl BasicAuth {
+    pub fn new() -> Self {
+        BasicAuth {
+            role: None,
+            policy: BasicAuth::policy(),
+        }
+    }
+
+    fn policy() -> Policy {
```

Comment:
only root policy should be defined

Reply:
this file is to be deleted. See comment earlier.